src/routes/docs/tooling/command-line/commands/+page.markdoc

```diff
@@ -4,65 +4,159 @@ title: Commands
 description: Learn about Appwrites CLI and the powerful, feature complete commands to manage Appwrite's auth, databases, functions, storage, and more.
 ---
 
-Other than commands to create and deploy databases, collections, functions, teams, and buckets, the Appwrite CLI can be used as a Server SDK as well. The Appwrite CLI has a command for every Server API endpoint.
+{% info title="CLI Version" %}
+All commands are compatible with the latest version of the CLI. We recommend running the [CLI on its latest version](/docs/tooling/command-line/installation#update-your-cli).
+{% /info %}
 
-Commands follows the following general syntax:
+
+Other than commands to create and push databases, collections, functions, messaging-topics, teams, and buckets, the Appwrite CLI can be used as a Server SDK as well. The Appwrite CLI has a command for every Server API endpoint.
+
+Commands generally follow the following syntax:
 
 ```sh
 appwrite [COMMAND] [OPTIONS]
 ```
 
-# List {% #list %}
+# Commands {% #commands %}
 
 Below is a list of the available commands in the Appwrite CLI. You can get more information on each command by running `appwrite [COMMAND] --help`.
 
+## General commands {% #general-commands %}
+
 {% table %}
 * Command
 * Description
 ---
-* `login`
-* The login command allows you to authenticate into the CLI. This command expects the console account that you use to log into the Appwrite Console.
+* `assistant`
+* The assistant command allows you to interact with the Appwrite Assistant AI.
 ---
-* `client`
+* `client [options]`
 * The client command allows you to configure your CLI.
 ---
-* `init`
-* The init command helps you initialize your Appwrite project, databases, collections, functions, teams, and buckets.
+* `locale`
+* The locale command allows you to customize your app based on your users' location.
+---
+* `console`
+* The console command gives you access to the APIs used by the Appwrite console.
+---
+* `graphql`
+* The graphql command allows you to query and mutate any resource type on your Appwrite server.
+---
+* `health`
+* The health command allows you to validate and monitor your Appwrite server's health.
+---
+* `migrations`
+* The migrations command allows you to migrate data between services.
+---
+* `vcs`
+* The vcs command allows you to interact with VCS providers and manage your code repositories.
```

Comment:
Let's leave out console, health, migrations, and vcs since we don't really want people to use those. That's why they're not publicly documented in the [API reference](https://appwrite.io/docs/references).